CHANGELOG.md

```diff
@@ -29,6 +29,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Delete OCDS and 360 logos. They are saved in cove/cove-ocds and cove-opendataservices-coop repositories.
 - Delete lib-cove-web/cove/templates/multi_index.html. Not being used (it was replaced by the static site at 
 cove-opendataservices-coop).
+- Add 'Co-operative' to 'built by'.
```

Comment:
This should be added under the "Unreleased" seciton at the top, since 0.15.0 is already released.

Reply:
Thanks @Bjwebb ! I have moved it.